dts/arm/nxp/nxp_rt11xx.dtsi

```diff
@@ -527,7 +527,7 @@
 			reg = <0x4018c000 0x4000>;
 			interrupts = <129 0>;
 
-			flexpwm1_pwm0: pwm0 {
+			flexpwm1_pwm0: flexpwm1_pwm0 {
```

Comment:
can you explain the reason for this change

Reply:
Yes, this is to match what was also changed in the [rt10xx.dtsi](https://github.com/zephyrproject-rtos/zephyr/commit/176d51555cdbe7da107d757cea9ed8d80e220512). The problem becomes that you can have multiple pwmx devices on same channel meaning that you cannot individually address them as expected or needed.